classes/requests/helpers/class-dintero-checkout-cart.php

```diff
@@ -381,14 +381,16 @@ public function get_shipping_item( $shipping_rate ) {
 		$shipping_cost = floatval( $shipping_rate->get_cost() );
 		$shipping_tax  = floatval( $shipping_rate->get_shipping_tax() );
 
+		$description = $this->get_shipping_method_option( $shipping_rate, 'dintero_description' );
```

Comment:
Suggestion: We have support for this in our [shipping package](https://github.com/krokedil/shipping/blob/master/src/ShippingRate.php#L85-L92) as well. With a small helper class to set/get a description on a shipping rate. This is used by for example our KSC plugin. Should we add support for that too while we are doing this anyway?

Also with this we currently only support the setting we add in the plugin directly. This wont add support if others add a description.